### PR TITLE
Extract getQueryString out of Operation template

### PR DESCRIPTION
**Pull request by fredells**
Part 2 of an effort to reduce the amount of generated code.

- extracts the getQueryString method that was common to all operations
- also removes an unused field on `selection`

### CODE REVIEW

File: Templates/Kotlin/GraphApi.stencil [hunk 1]
```diff
@@ -15,15 +15,25 @@ import java.lang.reflect.Type
 
 const val DEFINED_NULL_FLAG = "__DEFINED_NULL"
 
-interface SyrupOperation<T: Response> {
-    fun getQueryString(): String
+interface SyrupOperation<T : Response> {
+    val rawQueryString: String
+    fun getQueryString(): String {
```
Comment (jaredh) [inline Templates/Kotlin/GraphApi.stencil:20]: We should think of a more descriptive name to differentiate the two. Not really sure the difference between `getQueryString` and `rawQueryString` when they both return `String`.

Comment (jaredh) [inline Templates/Kotlin/GraphApi.stencil:20]: Suggest moving to all properties here too.

Comment (fredells) [reply Templates/Kotlin/GraphApi.stencil:20]: rawQueryString and networkQueryString?

Comment (fredells) [reply Templates/Kotlin/GraphApi.stencil:20]: to clarify, make getQueryString a computed property?

Comment (fredells) [reply Templates/Kotlin/GraphApi.stencil:20]: going to make this change too
https://github.com/Shopify/android/pull/14076/files#r419182367

Comment (jaredh) [reply Templates/Kotlin/GraphApi.stencil:20]: Maybe make it private and rename it `operationTemplate`, and `rawQueryString` -> `toSerializedOperation` or just `operation`. 

Misnomer using the term `query` here.

Comment (fredells) [reply Templates/Kotlin/GraphApi.stencil]: decided not to include these changes in this PR to avoid excessive refactoring

**Pull request merged**